firefox/janitor-git.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Firefox (git)",
+  "description": "The Firefox browser (git flavour)",
+  "icon": "https://janitor.technology/img/firefox.svg",
+  "docker": {
+    "image": "JanitorTechnology/firefox-git:base"
+  },
+  "ports": {
+    "22": {
+      "proxy": "none"
+    },
+    "8088": {
+      "proxy": "https",
+      "label": "VNC"
+    },
+    "8089": {
+      "proxy": "https",
+      "label": "Cloud9"
+    }
+  },
+  "previewUrls": {
+    "VNC": "https://${HOSTNAME}/${CONTAINER}/8089/vnc.html"
+  },
+  "scripts": {
+    "./mach build": "./mach build",
+    "./mach build faster": "./mach build faster",
+    "./mach build binaries": "./mach build binaries",
+    "./mach run": {
+      "cmd": "./mach run",
+      "openPreview": "VNC"
+    },
+    "./mach clobber": "./mach clobber",
+    "Send to code review": "git bz attach -e origin/master..HEAD",
+    "Update source code": "git fetch origin && git rebase origin/master"
```

Comment:
Nit: "Update source code" should probably be above "Send to code review" (I think it should always be the last script).